### ISSUE TITLE
ng-router

### ISSUE THREAD

**Issue by cgatian** [BeastCode/VSCode-Angular-TypeScript-Snippets#28]
snippet: `ng-router`

issue: Extra parenethsis at he end of the routes array

**Generated Code**
```
import { Routes } from '@angular/router';

import { HomeComponent } from './';
import { Name2Component } from './';
import { Name3Component } from './';
import { Name4Component } from './';
import { PageNotFoundComponent } from './';

const routes: Routes = [
  { path: '', component: HomeComponent },
  { path: 'path2', component: Name2Component },
  { path: 'path3', component: Name3Component },
  { path: 'path4', component: Name4Component },
  { path: '**', component: PageNotFoundComponent },

  //{ path: 'path/:routeParam', component: MyComponent },
  //{ path: 'staticPath', component: ... },
  //{ path: '**', component: ... },
  //{ path: 'oldPath', redirectTo: '/staticPath' },
  //{ path: ..., component: ..., data: { message: 'Custom' }
]);
```
 
Last line has `]);` which should be `];`

**Comment by cgatian** [comment]
This snippet should probably also `export` the `routes` array;

**Comment by BeastCode** [comment]
Fixed in v4.2.4

**Comment by cgatian** [comment]
Wow, that was.... fast 🚀

**Comment by BeastCode** [comment]
I took a look and found the same bug in two other places :( So decided to fix it right away.

If you plan on submitting a pull request just add a note in the description sand I'll leave them alone.

Thanks again!

**Comment by cgatian** [comment]
👍

**Comment by cgatian** [comment]
@BeastCode Suggestion to rename the exported module from `FeatureModule` to `FeatureRoutingModule`

This would align with the convention used in the [docs](https://angular.io/docs/ts/latest/guide/router.html)

```
@NgModule({
  imports: [RouterModule.forChild(routes)]
  exports: [RouterModule]
})
export class FeatureModule {}
```

**Comment by BeastCode** [comment]
Nice catch! fixed in v4.2.5
